modules/openapi-generator/src/main/resources/kotlin-spring/dataClassOptVar.mustache

```diff
@@ -1,4 +1,5 @@
 {{#useBeanValidation}}{{#required}}
         {{^isReadOnly}}@get:NotNull{{/isReadOnly}} {{/required}}{{>beanValidationModel}}{{/useBeanValidation}}{{#swaggerAnnotations}}
-        @ApiModelProperty({{#example}}example = "{{{example}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}{{#isReadOnly}}readOnly = {{{isReadOnly}}}, {{/isReadOnly}}value = "{{{description}}}"){{/swaggerAnnotations}}
+        @ApiModelProperty({{#example}}example = "{{{example}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}{{#isReadOnly}}readOnly = {{{isReadOnly}}}, {{/isReadOnly}}value = "{{{description}}}"){{/swaggerAnnotations}}{{#deprecated}}
+        @Deprecated(message=""){{/deprecated}}
```

Comment:
I think an empty message here will cause some code smells in static analysis (like Sonar). Should we provide a default message?

Reply:
Kotlin:`message` argument is required. 
```kotlin
public annotation class Deprecated(
    val message: String,
    val replaceWith: ReplaceWith = ReplaceWith(“”),
    val level: DeprecationLevel = DeprecationLevel.WARNING
)
```
We have to set empty message because `openapi` can't set "deprecated message".
Or set default message(for example "This property is required") 🤔

Reply:
We can later use vendor extension (not this PR) to achieve this, e.g. x-deprecated-message if there're demands from users to specify the deprecated message.

Reply:
@wing328 Thank you! 
I'll try it if I have time